include/dfrn.php

```diff
@@ -873,23 +948,28 @@ private function entry($doc, $type, $item, $owner, $comment = false, $cid = 0) {
 	 * @param bool $dissolve (to be documented)
 	 *
 	 * @return int Deliver status. -1 means an error.
+	 * @todo Add array type-hint for $owner, $contact
 	 */
 	public static function deliver($owner,$contact,$atom, $dissolve = false) {
 
 		$a = get_app();
 
 		$idtosend = $orig_id = (($contact['dfrn-id']) ? $contact['dfrn-id'] : $contact['issued-id']);
 
-		if($contact['duplex'] && $contact['dfrn-id'])
+		if ($contact['duplex'] && $contact['dfrn-id']) {
 			$idtosend = '0:' . $orig_id;
-		if($contact['duplex'] && $contact['issued-id'])
+		}
+		if ($contact['duplex'] && $contact['issued-id']) {
 			$idtosend = '1:' . $orig_id;
-
+		}
 
 		$rino = get_config('system','rino_encrypt');
 		$rino = intval($rino);
+
 		// use RINO1 if mcrypt isn't installed and RINO2 was selected
-		if ($rino==2 and !function_exists('mcrypt_create_iv')) $rino=1;
+		if ($rino==2 and !function_exists('mcrypt_create_iv')) {
```

Comment:
I would prefer to have a completely cleaned file in one task. But I don't see this as a blocker.